src/app/error.tsx

```
@@ -0,0 +1,25 @@
+'use client'
+
+import { Button } from '@/components/ui/button'
+import Image from 'next/image'
+import Link from 'next/link'
+
+const Error = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)